networking/private-cloud-function-from-onprem/main.tf

```diff
@@ -251,22 +239,21 @@ module "function-hello" {
 ###############################################################################
 
 module "bucket-functions" {
```

Comment:
You can use `bucket-config` in the function module to auto-create bucket, and skip the GCS module.